CreeDictionary/tests/conftest.py

```diff
@@ -51,6 +51,14 @@ def random_lemmas(draw) -> Wordform:
     return draw(sampled_from(list(lemmas)))
 
 
+@pytest.fixture(scope="session")
+def test_xml_dir() -> Path:
+    """
+    XML for the test database, with curated test data.
+    """
+    return shared_res_dir / "test_dictionaries"
```

Comment:
I wrote this when I was having difficulty figuring out the test data, but now it is unused. Consider deleting it.

Reply:
done